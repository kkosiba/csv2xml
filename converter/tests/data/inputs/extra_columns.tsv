H1	H2	H3	H4	H5
1_1	2_1	3_1	4_1	5_1
1_2	2_2	3_2	4_2	5_2	6_2	7_2	8_2
1_3	2_3	3_3	4_3	5_3	6_3	7_3	8_3
1_4	2_4	3_4	4_4	5_4	6_4	7_4	8_4	9_4	10_4
1_5	2_5	3_5	4_5	5_5
