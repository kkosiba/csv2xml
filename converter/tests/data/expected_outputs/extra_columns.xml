<?xml version="1.0"?>
<Document>
  <Row>
    <Field>
      <FieldName pos="1">H1</FieldName>
      <FieldContent pos="1">1_1</FieldContent>
    </Field>
    <Field>
      <FieldName pos="2">H2</FieldName>
      <FieldContent pos="2">2_1</FieldContent>
    </Field>
    <Field>
      <FieldName pos="3">H3</FieldName>
      <FieldContent pos="3">3_1</FieldContent>
    </Field>
    <Field>
      <FieldName pos="4">H4</FieldName>
      <FieldContent pos="4">4_1</FieldContent>
    </Field>
    <Field>
      <FieldName pos="5">H5</FieldName>
      <FieldContent pos="5">5_1</FieldContent>
    </Field>
    <Source nametype="leafnoext">extra_columns</Source>
    <Worksheet>extra_columns</Worksheet>
  </Row>
  <Row>
    <Field>
      <FieldName pos="1">H1</FieldName>
      <FieldContent pos="1">1_2</FieldContent>
    </Field>
    <Field>
      <FieldName pos="2">H2</FieldName>
      <FieldContent pos="2">2_2</FieldContent>
    </Field>
    <Field>
      <FieldName pos="3">H3</FieldName>
      <FieldContent pos="3">3_2</FieldContent>
    </Field>
    <Field>
      <FieldName pos="4">H4</FieldName>
      <FieldContent pos="4">4_2</FieldContent>
    </Field>
    <Field>
      <FieldName pos="5">H5</FieldName>
      <FieldContent pos="5">5_2</FieldContent>
    </Field>
    <Source nametype="leafnoext">extra_columns</Source>
    <Worksheet>extra_columns</Worksheet>
  </Row>
  <Row>
    <Field>
      <FieldName pos="1">H1</FieldName>
      <FieldContent pos="1">1_3</FieldContent>
    </Field>
    <Field>
      <FieldName pos="2">H2</FieldName>
      <FieldContent pos="2">2_3</FieldContent>
    </Field>
    <Field>
      <FieldName pos="3">H3</FieldName>
      <FieldContent pos="3">3_3</FieldContent>
    </Field>
    <Field>
      <FieldName pos="4">H4</FieldName>
      <FieldContent pos="4">4_3</FieldContent>
    </Field>
    <Field>
      <FieldName pos="5">H5</FieldName>
      <FieldContent pos="5">5_3</FieldContent>
    </Field>
    <Source nametype="leafnoext">extra_columns</Source>
    <Worksheet>extra_columns</Worksheet>
  </Row>
  <Row>
    <Field>
      <FieldName pos="1">H1</FieldName>
      <FieldContent pos="1">1_4</FieldContent>
    </Field>
    <Field>
      <FieldName pos="2">H2</FieldName>
      <FieldContent pos="2">2_4</FieldContent>
    </Field>
    <Field>
      <FieldName pos="3">H3</FieldName>
      <FieldContent pos="3">3_4</FieldContent>
    </Field>
    <Field>
      <FieldName pos="4">H4</FieldName>
      <FieldContent pos="4">4_4</FieldContent>
    </Field>
    <Field>
      <FieldName pos="5">H5</FieldName>
      <FieldContent pos="5">5_4</FieldContent>
    </Field>
    <Source nametype="leafnoext">extra_columns</Source>
    <Worksheet>extra_columns</Worksheet>
  </Row>
  <Row>
    <Field>
      <FieldName pos="1">H1</FieldName>
      <FieldContent pos="1">1_5</FieldContent>
    </Field>
    <Field>
      <FieldName pos="2">H2</FieldName>
      <FieldContent pos="2">2_5</FieldContent>
    </Field>
    <Field>
      <FieldName pos="3">H3</FieldName>
      <FieldContent pos="3">3_5</FieldContent>
    </Field>
    <Field>
      <FieldName pos="4">H4</FieldName>
      <FieldContent pos="4">4_5</FieldContent>
    </Field>
    <Field>
      <FieldName pos="5">H5</FieldName>
      <FieldContent pos="5">5_5</FieldContent>
    </Field>
    <Source nametype="leafnoext">extra_columns</Source>
    <Worksheet>extra_columns</Worksheet>
  </Row>
</Document>
